run.py

```diff
@@ -325,7 +325,11 @@ def calculate_average_read_length(file_path):
         weights = list(total_reads_dict.values())
 
         # Calculate the weighted average fraction of rRNA reads across all inputs in sample using numpy
-        weighted_rrna_fraction = np.average(fractions_rrna_in_subset, weights=weights)
+        if sum(total_reads_dict.values()) == 0:
```

Comment:
nit: this might would be slightly nicer to me as `sum(weights)`, since it's less repetitive